LJAPI.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+import os
+import requests
+import transaction
+
+
+from LearningJournal.models import Entry
+from sqlalchemy import engine_from_config
+from sqlalchemy.orm import scoped_session, sessionmaker
+from zope.sqlalchemy import ZopeTransactionExtension
+
+DBSession = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
+
+
+database_url = os.environ.get('DATABASE_URL', None)
+engine = engine_from_config({'sqlalchemy.url': database_url})
+DBSession.configure(bind=engine)
+
+json_entries = {}
+
+
+def get_json():
+    """Get and store journal json data."""
+    resp = requests.get('https://sea401d2.crisewing.com/api/export?apikey=' + os.environ.get('API_KEY'))
+
+    if resp.status_code == 200:
+        json_entries = resp.json()
+        return json_entries
```

Comment:
Requests provides a nice way to do this without checking first:

```python
resp.raise_for_status()
json_entries = resp.json
return json_entries
```

The call to `raise_for_status` will raise an error if a 400+ response code is returned.  If the response was 200 or 300 level, then it does nothing.